src/+otp/+arenstorf/ArenstorfParameters.m

```diff
@@ -0,0 +1,10 @@
+classdef ArenstorfParameters
+    %ARENSTORFPARAMETERS User-configurable parameters for the Arenstorf problem
+    %
+    %   See also otp.arenstorf.ArenstorfProblem
+    
+    properties
+        %MU The mass of one body, while the other body has mass 1-Mu
+        Mu %MATLAB ONLY: (1, 1) {mustBeReal, mustBeFinite}

```

Comment:
Does "mustBeReal" work with symbolic? it looks like you are trying to support it with the classes, but I am not sure that the matlab code here will work with it.

Reply:
`mustBeReal` works with symbolic and vpa in matlab but not octave.  That seems like an octave bug, so I say we keep the validation as is.